app/javascript/menu/dropdowns.js

```diff
@@ -7,7 +7,7 @@ document.addEventListener('turbolinks:load', () => {
 
   dropdowns.forEach(function (dropdown) {
     dropdown.addEventListener('click', function (e) {
-      e.stopPropagation()
+      // e.stopPropagation()
```

Comment:
Will we need this code? When and in wick circumstances?

Reply:
It's really no need. I'd like to leave as a result of battles with UI(dropdown menu+hiding+speed of scroll). I think it’s gonna be okay